notebooks/Paper_replication.ipynb

```diff
@@ -46,12 +46,12 @@
   },
```

Comment:
<p>If this error is fixed you need to re-run the notebook</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/laos-climate-change/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done